stps/sig-network/ip-request-stp.md

```diff
@@ -0,0 +1,216 @@
+# Openshift-virtualization-tests Test plan
+
+## **Request a specific IP for a VM in UDN through a third party integration - Quality Engineering Plan**
+
+### **Metadata & Tracking**
+
+| Field                   | Details                                                                       |
+|:----------------------- |:----------------------------------------------------------------------------- |
+| **Enhancement(s)**      | [As close as possible to VEP](https://access.redhat.com/solutions/7133388)    |
+| **Feature in Jira**     | https://issues.redhat.com/browse/CNV-67524                                    |
+| **Jira Tracking**       | https://issues.redhat.com/browse/CNV-70089                                    |
+| **QE Owner(s)**         | Yoss Segev (ysegev@redhat.com)                                                |
+| **Owning SIG**          | sig-network                                                                   |
+| **Participating SIGs**  | sig-network                                                                   |
+| **Current Status**      | Draft                                                                         |
+
+---
+
+### **I. Motivation and Requirements Review (QE Review Guidelines)**
+
+This section documents the mandatory QE review process. The goal is to understand the feature's value, technology, and testability prior to formal test planning.
+
+#### **1. Requirement & User Story Review Checklist**
+
+| Check                                   | Done  | Details/Notes                                                                                                                                                                            | Comments  |
```

Comment:
the details need to be updated with the feature-related data, e.g
`Understand Value` -> enable customers to integrate with external IPAM systems